<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fParamsDlg</class>
 <widget class="QDialog" name="fParamsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fParamsDlg</string>
  </property>
  <widget class="QDoubleSpinBox" name="widthSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="widthLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>width</string>
   </property>
  </widget>
  <widget class="QLabel" name="heightLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>height</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="heightSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseMesh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="meshLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>mesh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseFrames">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="framesLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>frames</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseSupportingAreas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="supportingAreasLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>supporting areas</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>fParamsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>fParamsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
